rpc/walletrpc/methods.go

```diff
@@ -97,13 +61,12 @@ func ListAccounts(icmd interface{}, w *wallet.Wallet) (interface{}, error) {
 	return accountBalances, nil
 }
 
-// getNewAddress handles a getnewaddress request by returning a new
+// getNewAddress handles a GetNewAddress request by returning a new
 // address for an account.  If the account does not exist an appropiate
 // error is returned.
-// TODO: Follow BIP 0044 and warn if number of unused addresses exceeds
 // the gap limit.
-func GetNewAddress(icmd interface{}, w *wallet.Wallet) (interface{}, error) {
-	cmd := icmd.(*qitmeerjson.GetNewAddressCmd)
+func GetNewAddress(iCmd interface{}, w *wallet.Wallet) (interface{}, error) {
```

Comment:
need not do this. 
`func getNewAddress(icmd interface{}, w *wallet.Wallet) (interface{}, error) {` is ok